modules/ROOT/pages/ch2-clustering.adoc

```diff
@@ -0,0 +1,125 @@
+= Clustering
+
+*THIS IS CH 1.0 PLACEHOLDER CONTENT* 
+
+*Removed persistent queues; point to Anypoint MQ instead.*
```

Comment:
In CH2.0 mule clustering capabilities are provided by Hazelcast, not sure how Anypoint MQ comes into play here. Maybe @jeff-1amstudios can help clarify?